### DISCUSSION TITLE
🔥 Handling Keyboard and Mouse events

### DISCUSSION THREAD

**Discussion by yandeu** [enable3d/enable3d#126] [Show and tell]
I just created 2 awesome packages for handling Mouse, Touch, Pointer and Keyboard events:

- [🖱️ Tap](https://github.com/yandeu/tap)
- [⌨️ Keyboard](https://github.com/yandeu/keyboard)

You can use them in any project.

They will be used internally by some enable3d classes, but will not be exposed by the framework.

**Comment by N8python** [comment]
Epic, love the ecosystem forming around `enable3d`.

**Comment by rabbitLove520** [comment]
thks a lot.
